internal/cluster/transition.go

```diff
@@ -72,6 +72,7 @@ type TransitionHandler interface {
 	PostRefreshFinalizingStageSoftTimedOut(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) error
 	SoftTimeoutsEnabled(_ stateswitch.StateSwitch, _ stateswitch.TransitionArgs) (bool, error)
 	FinalizingStageTimeoutMinutes(sCluster *stateCluster) interface{}
+	InstallationTimeoutMinutes(_ *stateCluster) interface{}
```

Comment:
why we need this function?

Reply:
For formatting the status info

Reply:
The installation timeout member is a data member of `transitionHandler`, and the statemachine accesses the interface (not the object itself). Therefore it needs to be part of the interface too.